### ISSUE TITLE
Release 2.14.0 breaks cluster_ca (and https cluster_urls)

### ISSUE THREAD

**Issue by calvinmorrow** [linkyard/concourse-helm-resource#119]
```
  - name: helm
    type: helm
    source:
      cluster_url: https://kubernetes.example.com
      cluster_ca: ((kubernetes/dev.ca))
      token: ((kubernetes/dev.token))
      tillerless: true
```
Results in:
```
resource script '/opt/resource/check []' failed: exit status 1

stderr:
Initializing kubectl...
User "admin" set.
Context "default" modified.
Switched to context "default".
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.2", GitCommit:"66049e3b21efe110454d67df4fa62b08ea79a19b", GitTreeState:"clean", BuildDate:"2019-05-16T16:23:09Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```

This seems to be because https://github.com/linkyard/concourse-helm-resource/blame/015f10bc6da038fbaa6a3fc8ea3208ae549d1a09/assets/common.sh#L25 was changed/removed with the recent kubeconfig changes.

**Comment by nimakaviani** [comment]
i am having the same problem.

**Comment by nimakaviani** [comment]
I believe 2.13.1 is also broken. I had to rollback to `2.13.0` for it to successfully work with tiller.

**Comment by msiegenthaler** [comment]
I am seeing the same, but 2.13.1-tillerless still works.
2.13.1 should not be affected anyway, that was before even the tillerless

**Comment by msiegenthaler** [comment]
Should be fixed now in `2.14.0-1` (and `latest`). Sorry about that, I missed that case in the review :(
